integration/aggregate_documents_compat_test.go

```diff
@@ -2156,14 +2155,6 @@ func TestAggregateCompatProject(t *testing.T) {
 			resultType:     emptyResult,
 			failsForSQLite: "https://github.com/FerretDB/FerretDB/issues/3148",
 		},
-		"SumValue": {
```

Comment:
Duplicate of `TestAggregateCompatProjectSum/Value` https://github.com/FerretDB/FerretDB/pull/3180/files?diff=split&w=0#diff-2ac27c261718f53527a14f93e58a43aad7ece8bcbbe7036fe67a9ced284eb9dbR2168